codebuild/bin/apache2/ports.conf

```diff
@@ -2,6 +2,6 @@
 # have to change the VirtualHost statement in
 # /etc/apache2/sites-enabled/000-default.conf
 
-Listen 11111
+Listen 7777
```

Comment:
I'd put a comment here noting that this should be below the integration test framework-- make sure someone else doesn't make the same mistake in the future :)

Reply:
Added!